install_scripts/templates/kubernetes/init.sh

```diff
@@ -596,7 +596,7 @@ if [ "$NO_PROXY" != "1" ]; then
         discoverProxy
     fi
 
-    if [ -z "$PROXY_ADDRESS" ]; then
+    if [ -z "$PROXY_ADDRESS" ] && [ "$AIRGAP" -ne "1" ]; then
```

Comment:
We generally use string equality tests rather than numeric equality tests because some of these params are taken directly from the querystring.